src/test/java/BaseTest.java

```diff
@@ -1,12 +1,111 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
-import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.*;
 
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.time.Duration;
+import java.util.HashMap;
 
 public class BaseTest {
 
-    @BeforeSuite
-    static void setupClass() {
-        WebDriverManager.chromedriver().setup();
+    private static final ThreadLocal<WebDriver> THREAD_DRIVER = new ThreadLocal<>();
+    protected static WebDriverWait wait;
+    protected String password = "";
+    protected String email = "";
+    protected String homeUrl = "";
+    private final static int TIME = 8; // time to set up implicitlyWait for the browser
+
+
+//    @BeforeSuite
+//    protected static void setUpClass () {
+//        WebDriverManager.chromedriver().setup();
+//    }
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

Reply:
Oh, it was just something that I used way earlier and just left there just in case, but obviously should have deleted it